cognite/powerops/utils/cdf_auth.py

```diff
@@ -4,21 +4,36 @@
 from cognite.client.credentials import OAuthClientCredentials, OAuthDeviceCode
 
 
+class BootstrapConfigError(Exception):
+    """Exception raised for config validation
+
+    Attributes:
+        message -- explanation of the error
+    """
+
+    def __init__(self, message: str):
+        self.message = message
+        super().__init__(self.message)
+
+
 def get_client(parameters: dict) -> CogniteClient:
     client_name = "power-ops-bootstrap"
 
-    client_id = parameters.get("CLIENT_ID")
     cluster = parameters.get("CDF_CLUSTER")
-    project = parameters.get("COGNITE_PROJECT")
     tenant_id = parameters.get("TENANT_ID")
-    client_secret_env = parameters.get("CLIENT_SECRET_ENV")
 
     base_url = f"https://{cluster}.cognitedata.com"
     scopes = [f"https://{cluster}.cognitedata.com/.default"]
 
-    client_secret = os.getenv(client_secret_env) if client_secret_env else None
+    project = parameters.get("COGNITE_PROJECT") if os.getenv("PYTEST") else os.getenv("COGNITE_PROJECT")
```

Comment:
Keeping these two to be read from cdf.yaml so that pytest can run in github - or they will have to be passed in as env variables in the pytest workflow